src/bin/pg_dump/pg_dump.c

```diff
@@ -8562,6 +8562,21 @@ getCasts(Archive *fout, int *numCasts)
 		/* Decide whether we want to dump it */
 		selectDumpableCast(&(castinfo[i]), fout);
 
+		/*
+		 * Do not dump following unused CASTS:
```

Comment:
I think we should create a separate file for babel dump and put logic in there. We can call that function from here.

Reply:
Done!